src/components/ModalWindow/ModalWindow.module.scss

```diff
@@ -3,9 +3,14 @@
 }
 
 input:invalid {
-  border: 1px solig rgb(255, 0, 0) !important;
+  border: 1px solig rgb(211, 3, 3) !important;
 }
 
 input {
   border: 1px solig rgb(0, 0, 0) !important;
+}
+
+.formControl {
+  background-color: #ff000021 !important;
+  border: 2px solid #250707 !important;
```

Comment:
GitHub needs empty line at the end of each file you want to add

Reply:
yes, for some reason my prittier removes last empty line in styles, i will find where it changes

Reply:
I added ' "files.insertFinalNewline": true ' to settings.json , so now it work as you mentioned